MetaMask/metamask-extension
# fix: add metrics event to sign controller

## Explanation



`metricsEvent` callback is missing while initialising SignController, and leads bug on close Sign approvals.

## Manual Testing Steps



- Go [test dapp](https://metamask.github.io/test-dapp/)
- Try "Personal Sign"
- Close approval window (error will be thrown and approval wont be rejected)
- See notification persists because of error 

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
